src/triggers/CBLlot.trigger

```diff
@@ -0,0 +1,18 @@
+trigger CBLlot on CBL_Lot__C(after insert){
+  for (CBL_Lot__C newLot : Trigger.new) {
+    if (newLot.CBL2_Lot__c <> null) {
+      for(CBL2_Lot__c cbl2lot : [SELECT lotArea__c, lotName__c, lotAreaUnits__c, lotCode__c, Lot_Code_Guat__c, lotCoffeeVariety__c, Lot_Latitude__c, Lot_Longitude__c, lotNumberOfPlants__c, lotPlantingDate__c, Submission__c FROM CBL2_Lot__c WHERE Id = :newLot.CBL2_Lot__c]){ 
```

Comment:
This isn't bulk-safe. Will cause a problem when a batch of BBL_Lot__c objects are inserted. Should do one soql for the entire batch.